ledger/daml-on-sql/src/main/scala/on/sql/Cli.scala

```diff
@@ -25,6 +27,15 @@ private[sql] final class Cli(override val defaultConfig: SandboxConfig = Default
       .text("Allows development versions of DAML-LF language and transaction format.")
       .hidden()
 
+    parser
```

Comment:
We have almost the same logic 3 times now including 3 separate tests. For the participant config, the JSON API CLI and the DAML on SQL CLI. Maybe worth thinking about whether we can factor this out. (totally fine to not do this in this PR)

Reply:
![Make it so.](https://media.giphy.com/media/bKnEnd65zqxfq/giphy.gif)

Reply:
I'd rather do refactorings in separate PRs indeed.